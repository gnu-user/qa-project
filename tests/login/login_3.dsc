TITLE
    Login User Authentication
DESCRIPTION
    Attempt to initiate the login transaction by entering “login” into the 
    input, and then read in the current user accounts file. Verify that the 
    output contains a prompt for entry of a username. Enter a valid username 
    and verify that it is accepted. 
DEPENDENCIES
