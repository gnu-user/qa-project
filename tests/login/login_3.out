Welcome.
> 
Enter username: 
Username accepted.